src/kubernetes-ingress-controller/guides/using-kongplugin-resource.md

```diff
@@ -306,8 +369,13 @@ Access-Control-Allow-Credentials: true
 X-Kong-Upstream-Latency: 2
 X-Kong-Proxy-Latency: 1
 Via: kong/1.2.1
+```
+
+```sh
```

Comment:
I wonder if having the exact say command tested on both `foo` and `baz` is useful here, or if it just creates clutter. If you think it makes sense to leave it, then preface it with:
```suggestion
Then use the API key with `/foo`:
```sh
```